### PR TITLE
Adds A.com georouter modal to express

### PR DESCRIPTION
**Pull request by jsandland**
Adds the A.com georouter block / modal to express

Resolves: [MWPW-154379](https://jira.corp.adobe.com/browse/MWPW-154379)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/in/express/
- After: https://georouter--express--adobecom.hlx.page/in/express/

### CODE REVIEW

File: express/scripts/scripts.js [hunk 1]
```diff
@@ -12,26 +12,25 @@ import {
...
-  it: { ietf: 'it-IT', tk: 'bbf5pok.css' },
+  '': { ietf: 'en-US', tk: 'hah7vzn.css' },
+  br: { ietf: 'pt-BR', tk: 'hah7vzn.css' },
+  cn: { ietf: 'zh-CN', tk: 'qxw8hzm' },
+  de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
+  dk: { ietf: 'da-DK', tk: 'qxw8hzm.css' },
+  es: { ietf: 'es-ES', tk: 'hah7vzn.css' },
+  fi: { ietf: 'fi-FI', tk: 'qxw8hzm.css' },
+  fr: { ietf: 'fr-FR', tk: 'hah7vzn.css' },
+  id_en: { ietf: 'en', tk: 'hah7vzn.css' },
```
Comment (vhargrave) [inline express/scripts/scripts.js:23]: i don't think this locale id_en needs to be added, because there's no content being created under that url.

File: .eslintrc.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ module.exports = {
     // allow reassigning param
     'no-param-reassign': [2, { props: false }],
     'linebreak-style': ['error', 'unix'],
+    'no-await-in-loop': 0,
+    'no-return-assign': 'off',
```
Comment (vhargrave) [inline .eslintrc.js:13]: could you make this the same as milo actually , so we're aligned ? - `'no-return-assign': ['error', 'except-parens'].`
I'd find it personally a bit confusing to see an assignment on the same line as a return statement

**Review by vhargrave — CHANGES_REQUESTED** [review]
I like how clean the copy paste is . I'd recommend checking out the test urls you linked with something like ?akamaiLocale=CH in your URL or ?akamaiLocale=DE
For countries that have more than one locale, the georouting looks broken and things like the flags are missing. 
In addition, I don't even think that we want the georouting to even appear for countries like Switzerland, because we don't have an experience prepared for CH, but maybe we want to take users to the German experience. This is technically an authoring decision, but since georouting is so new to the authors I think they'll need dev support to get this configured right.

**Comment by jsandland** [comment]
@vhargrave Regading the missing flags: 
I see a reference to the flags like this in milo:
```const flagFile = locale.globeGrid?.toLowerCase().trim() === 'on' ? 'globe-grid.png' : `flag-${locale.geo.replace('_', '-')}.svg`;```
Do those svg files live in the cloud?

**Comment by jsandland** [comment]
> For countries that have more than one locale, the georouting looks broken...
I tested every `locale` in `scripts.js` and observed georouting working for every entry. I have removed `Id_en` as requested so did not test that one. However, I also tested every locale with the akamaiLocale query param and for every one of those cases the `georouter` failed to appear. Is this what you mean by broken? What is the expected behavior when the `akamaiLocale` is included? Doesn't that force akamai to serve a country specific cached page?

File: express/scripts/scripts.js [hunk 3]
```diff
@@ -12,26 +12,24 @@ import {
...
 const locales = {
-  '': { ietf: 'en-US', tk: 'jdq5hay.css' },
-  br: { ietf: 'pt-BR', tk: 'inq1xob.css' },
-  cn: { ietf: 'zh-Hans-CN', tk: 'puu3xkp' },
-  de: { ietf: 'de-DE', tk: 'vin7zsi.css' },
-  dk: { ietf: 'da-DK', tk: 'aaz7dvd.css' },
-  es: { ietf: 'es-ES', tk: 'oln4yqj.css' },
-  fi: { ietf: 'fi-FI', tk: 'aaz7dvd.css' },
-  fr: { ietf: 'fr-FR', tk: 'vrk5vyv.css' },
-  gb: { ietf: 'en-GB', tk: 'pps7abe.css' },
```
Comment (vhargrave) [inline express/scripts/scripts.js:23]: why remove the gb locale?

**Review by vhargrave — COMMENTED** [review]
Regarding what we've discussed yesterday, I've thought about it more and I think the best approach for now is to only offer georouting for the locales that we have. 
This means offering a georouting modal for German , French, US, Indian, UK, users etc. 
But not offering a modal to take the user to the German experience if we detect they're in Switzerland. 
You'll probably want to double check if this is ok, but if we do want to start offering support so Georouting appears for Switzerland , then we'll need to have a proper discussion with the georouting PdMs on requirements. 

So in summary as long as we only have entries for the locales we have in the georoutingv2.xlsx file we should be good. 

Asides from that I found two things that need to be changed, and then we should slowly be good.
